arm-network/2016-12-01/swagger/serviceCommunity.json

```diff
@@ -0,0 +1,153 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2016-12-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/serviceCommunities": {
+      "get": {
+        "tags": [
+          "ServiceCommunities"
+        ],
+        "operationId": "ServiceCommunities_List",
+        "x-ms-examples": {
+          "ServiceCommunityList": { "$ref": "../examples/ServiceCommunityList.json" }
+        },
+        "description": "Gets all the available service communities.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ServiceCommunity resources.",
+            "schema": {
+              "$ref": "#/definitions/ServiceCommunityListResult"
+            }
+          }
```

Comment:
```
{ validityStatus: false,
  operations: 
   { ServiceCommunities_List: 
      { 'x-ms-examples': 
         { scenarios: 
            { ServiceCommunityList: 
               { isValid: false,
                 request: 
                  { isValid: true,
                    result: 'Request parameters for x-ms-example "ServiceCommunityList" in operation "ServiceCommunities_List" is valid.' },
                 responses: 
                  { '200': 
                     { isValid: false,
                       error: 
                        { code: 'RESPONSE_VALIDATION_ERROR',
                          message: 'Found errors in validating the response with statusCode "200" for x-ms-example "ServiceCommunityList" in operation "ServiceCommunities_List".',
                          innerErrors: 
                           [ { code: 'INVALID_RESPONSE_BODY',
                               errors: 
                                [ { code: 'OBJECT_ADDITIONAL_PROPERTIES',
                                    params: [ [ 'etag' ] ],
                                    message: 'Additional properties not allowed: etag',
                                    path: [ 'value', '1' ],
                                    description: 'Service Community Properties.' },
                                  { code: 'OBJECT_ADDITIONAL_PROPERTIES',
                                    params: [ [ 'state' ] ],
                                    message: 'Additional properties not allowed: state',
                                    path: [ 'value', '1', 'properties' ],
                                    description: 'Properties of Service Community.' },
                                  { code: 'OBJECT_ADDITIONAL_PROPERTIES',
                                    params: [ [ 'communityValue' ] ],
                                    message: 'Additional properties not allowed: communityValue',
                                    path: [ 'value', '1', 'properties', 'communities', '0' ],
                                    description: 'Contains bgp community information offered in Service Community resources.' },
                                  { code: 'OBJECT_ADDITIONAL_PROPERTIES',
                                    params: [ [ 'etag' ] ],
                                    message: 'Additional properties not allowed: etag',
                                    path: [ 'value', '0' ],
                                    description: 'Service Community Properties.' },
                                  { code: 'OBJECT_ADDITIONAL_PROPERTIES',
                                    params: [ [ 'state' ] ],
                                    message: 'Additional properties not allowed: state',
                                    path: [ 'value', '0', 'properties' ],
                                    description: 'Properties of Service Community.' },
                                  { code: 'OBJECT_ADDITIONAL_PROPERTIES',
                                    params: [ [ 'communityValue' ] ],
                                    message: 'Additional properties not allowed: communityValue',
                                    path: [ 'value', '0', 'properties', 'communities', '0' ],
                                    description: 'Contains bgp community information offered in Service Community resources.' } ],
                               message: 'Invalid body: Value failed JSON Schema validation',
                               path: [] } ] } } } } } } } } }
```

The examples in the "value" array of the response have properties like 
- "etag"
- "properties.state"

These properties are not defined in the swagger spec. Either the example is correct or the spec. Based on the nature of examples given , it looks like they are correct and the spec needs to be fixed.

- Another issue that is found:
"properties.communities[0].communityValue" is present in the example but not found in the spec. Turns out the property name was **mistyped**. The spec says the name is: - ["communityVa**ul**e"](https://github.com/Azure/azure-rest-api-specs/pull/855/files#diff-1a6ffa94255d7b1284acb32418b7c63aR80) in the spec instead of "communityVal**u**e"